elytron/src/test/resources/org/wildfly/extension/elytron/tls-test.xml

```diff
@@ -23,21 +23,25 @@
     </credential-stores>
     <tls>
         <key-stores>
-            <key-store name="FireflyKeystore" type="JKS">
-                <file path="firefly.keystore" relative-to="jboss.server.config.dir"/>
+            <key-store name="FireflyKeystore" >
                 <credential-reference store="test" alias="master-password-alias"/>
+                <implementation type="JKS" />
```

Comment:
I think we need to be careful if anything could end up with defaults defined for the attributes (we may be Ok with the ones you identify so far) - if attributes have defaults defined I don't think it makes sense for them to be represented in a child element.